src/components/navbar/search-area/SearchArea.js

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import './SearchArea.css';
+import {Link} from "react-router-dom";
+
+class SearchArea extends Component {
+
+  state = {
+    isGenreSelected: false,
+    isTitleSelected: true,
+    searchText: ""
+  };
+
+  render() {
+    return (
+      <div className="row mt-1 w-100">
+        <div className="col">
+          <span className="pink font-weight-bold">netflixroulette</span>
+        </div>
+        <div className="col-12 mt-5 white font-weight-bold"><span>FIND YOUR MOVIE</span></div>
+        <div className="col-12">
+          <form className="col-12 form-inline my-2 my-lg-0 pl-0">
+            <input
+              className="form-control mr-sm-2 white search-aria"
+              type="text"
+              placeholder="Search"
+              aria-label="Search"
+              value={this.state.searchText}
+              onChange={event => this.setState({searchText: event.target.value})}
+            />
+          </form>
+        </div>
+
+        <div className="col-4 mt-3 mb-4 ">
+          <span className="mr-3 white font-weight-bold">SEARCH BY</span>
+          <button
+            className={"btn my-2 my-sm-0 white " + (this.state.isTitleSelected ? "button-pink" : "button-black")}
+            type="button" onClick={event => this.searchByTitleClick(event)}
+          >
+            TITLE
+          </button>
+          <button
+            className={"btn my-2 ml-2 my-sm-0 white " + (this.state.isGenreSelected ? "button-pink" : "button-black")}
+            type="button" onClick={event => this.searchByGenreClick(event)}
+          >
+            GENRE
+          </button>
+        </div>
+        <div className="col-2 mt-3 mb-4 ml-auto search-button">
+          <Link to={this.getSearchURL()}>
```

Comment:
It's too  (i mean link to and button onClick)... Please, use link properties or button properties, but not both